run-tests.sh

```diff
@@ -79,11 +78,18 @@ function run_tests() {
     echo
     echo
 
+    # If TEST_NAMESPACE is not set (the default), we run 8 jobs in parallel
+    # otherwise, if it is set we can only run 1 job at a time
+    BATS_JOBS_PARAMS=""
+    if [[ -z "$TEST_NAMESPACE" ]]; then
```

Comment:
I suspect Shellcheck won't like undefined usage - you should default it to an empty string. You are actually checking it is zero-length, not that it is unset.

```suggestion
    if [[ -z "${TEST_NAMESPACE:-}" ]]; then
```